sched/hrtimer/hrtimer.c

```diff
@@ -49,30 +49,21 @@ static inline_function void hrtimer_reprogram(FAR hrtimer_queue_t *queue,
                                               uint64_t next_expired)
 {
 #ifdef CONFIG_SCHED_TICKLESS
-#  ifdef CONFIG_SCHED_TICKLESS_ALARM
-#    ifdef CONFIG_SCHED_TICKLESS_TICK_ARGUMENT
-  up_alarm_tick_start(div_const_roundup(next_expired, NSEC_PER_TICK));
-#    else
   struct timespec ts;
+#  ifdef CONFIG_SCHED_TICKLESS_ALARM
```

Comment:
merge to https://github.com/apache/nuttx/pull/17675/changes/e72a975db2c7cc024cba67a9ebdce20321fdba0a